chaos_genius/alerts/slack.py

```diff
@@ -192,7 +192,7 @@ def _format_slack_anomalies(
 
         out += (
             f"- *{kpi_name_link}* changed to "
-            f"*{point.y}* (*{point.change_message}*) "
+            f"*{point.y}* (*{point.formatted_change_percent}*) "
```

Comment:
can we have explicit string concatenation here, or may in a single commit we can do it for whole code base in that case we might need to add a TODO reminder

Reply:
i'll do it here for now